app/Http/Controllers/LikeController.php

```diff
@@ -27,25 +28,24 @@ public function getUserPostLike($user_id, $post_id): bool {
      * @param  \Illuminate\Http\Request  $request
      * @return \Illuminate\Http\Response
      */
-    public function store(Request $request)
+    public function store(Request $request): Response
     {
         $getLike = $this->getUserPostLike($request->user_id, $request->post_id);
```

Comment:
Could we rename this variable something more descriptive? If it's an instance of a Like, let's just call it `$like` - `get` would usually prefix a method name, not a variable name

Reply:
This was fixed already.